src/components/hero/AcceptDemoInvitation.tsx

```diff
@@ -32,7 +32,7 @@ function AcceptDemoInvitation({
 
     const { jobStatusPoller } = useJobStatusPoller();
 
-    const mutateAuthRoles = useEntitiesStore_mutate();
+    const mutateAuthRoles = useEntitiesStore((state) => state.mutate);
```

Comment:
Won't always remove the "mediator hooks" but this store is used in so few spots it seems safe to do it.